ansible/roles/clickhouse_proxy/tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Flush all handlers now
+  meta: flush_handlers
```

Comment:
It may be worth adding a brief comment here about the issue with the ordering of the handlers and link to the comment thread with your analysis.